bin/s2nc.c

```diff
@@ -93,6 +94,9 @@ void usage()
                     "    By default, the client will generate keyshares for all curves present in the ecc_preferences list.\n");
     fprintf(stderr, "  -L --key-log <path>\n");
     fprintf(stderr, "    Enable NSS key logging into the provided path\n");
+    fprintf(stderr, "  --psk <psk-identity, psk-secret, psk-hmac-alg> \n"
+                    "    A comma separated list of psk paramaters specified in an order namely, psk_identity, psk_secret and psk_hmac_alg.\n"
+                    "    Ex: --psk psk_id,psk_secret,S2N_PSK_HMAC_SHA256 --psk shared_id,shared_secret,S2N_PSK_HMAC_SHA384 \n");
```

Comment:
Nit: Are there spaces allowed between the identity, secret and hmac? I can't tell from this description